checker/bin-devel/build.sh

```diff
@@ -96,4 +96,6 @@ echo "... done: (cd ../stubparser/ && ./.build-without-test.sh)"
 echo "running \"./gradlew assemble\" for checker-framework"
 ./gradlew assemble --console=plain --warning-mode=all -s -Dorg.gradle.internal.http.socketTimeout=60000 -Dorg.gradle.internal.http.connectionTimeout=60000
 
+(cd "$CHECKERFRAMEWORK" && ./gradlew getDoLikeJavac -q)
```

Comment:
Did you put this line here so that `getDoLikeJavac` is run whenever the Checker Framework is built? I think instead of this, `assemble` should depend on `getDoLikeJavac`.  (If not, then when do you want it to run?)

Reply:
`getDoLikeJavac` doesn't *have* to be run every time the Checker Framework is built.  (But maybe doing so would be good practice?)  But it is essential that it appears in the distribution.

I think your suggestion is a good one; let's go with that.  Thanks!